### PR TITLE
Simple client demo

### PR DESCRIPTION
**Pull request by CriesofCarrots**
Toward #343 
The user interaction could be automated, if that type of demo is preferred.

This does not seem to work with testnet.solana.com at present, but does work with a local leader/drone.

### CODE REVIEW

**Comment by mvines** [comment]
Neat, can you please add a simple-client-demo entry to snapcraft.yaml too, like https://github.com/solana-labs/solana/blob/master/snap/snapcraft.yaml#L48

File: multinode-demo/simple-client.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# usage: $0 <rsync network path to solana repo on leader machine> <number of nodes in the network???>"
```
Comment (mvines) [inline multinode-demo/simple-client.sh]: Remove “number of nodes in the network” argument documentation?

**Comment by garious** [comment]
Let's dumb it down. No Rayon, no GenKeys, pay provided amount to provided pubkey from provided private key. Formats for keys need to be discussed, but just serde JSON will be fine to get this PR merged. Feel free to pipe in a renamed mint.json (id.json?) for that private key. See solana-genesis for that code.

**Comment by garious** [comment]
And can you rename to `wallet-demo`?

**Comment by garious** [comment]
Everything I requested can come in as a separate PR. Merging to ensure you don't end up in rebase hell.

**Comment by CriesofCarrots** [comment]
Thanks, @garious !
